iOS/JSQMessagesViewController/nuget/Xamarin.JSQMessagesViewController.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.JSQMessagesViewController</id>
+    <title>JSQMessagesViewController for Xamarin.iOS</title>
+    <version>1.11.7</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>An elegant messages UI library for iOS.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://components.xamarin.com/view/jsqmessagesviewcontroller</projectUrl>
```

Comment:
Use FWLinks. Point to Github and move license.md file from components folder to root of the project (`iOS/JSQMessagesViewController`)